nss-day-cohort-43/nuttyshell-the-coven-code
# fixed logout doubling up

# Description

- nutshell was being called on registerForm to kickoff the happenings, but it was called without an argument. that is now fixed and the correct welcome messages render with their owner

- logout button appears when there is an activeUser

- fixed logout doubling up bug. 

- also added divs for styling purposes around welcome greetings and logout button

Fixes # (logout button doubling when new user registers)
## Type of change


# Testing Instructions
Begin with clear sessionStorage. (1)Login as return user (2)logout (3)without a refresh, register a new user (4) confirm there is one logout button (5) make sure the right message is displayed for the current activeUser
# Checklist:
